backend/routes/uploadRoutes.js

```diff
@@ -28,7 +30,13 @@ const fileFilter = (req, file, cb) => {
 
 const upload = multer({ storage, fileFilter }).single('image');
 
-router.post('/', upload, (req, res) => {
+const validator = {
+  upload: [
+    check('file').notEmpty().withMessage("Invalid file")
```

Comment:
Please fix the image upload validation, it's showing errors even when I upload a valid image.

Reply:
fixed